game/src/ecs/items/Item.java

```diff
@@ -0,0 +1,118 @@
+package ecs.items;
+
+import ecs.components.AnimationComponent;
+import ecs.components.HitboxComponent;
+import ecs.components.PositionComponent;
+import ecs.entities.Entity;
+import graphic.Animation;
+import java.util.List;
+import mydungeon.ECS;
+import tools.Point;
+
+public abstract class Item {
+    public static final List<String> missingTexture = List.of("animation/missingTexture.png");
+
+    public static final String DefaultDescription = "Default Description.";
```

Comment:
schreibt man in Java final/default Attribute mit führenden Großbuchstaben?

Reply:
Eigentlich schreibt man sie wie folgt: DEFAULT_DESCRIPTION mein Fehler wird gleich behoben. Habe heute erstmal alle Merge und Rebases gemacht. Da ich mir einen temporären Master gebaut habe, damit ich schonmal weiter machen konnte bevor z.b. das Lebenssystem im Master gemerged wurde.

Reply:
> Da ich mir einen temporären Master gebaut habe, damit ich schonmal weiter machen konnte bevor z.b. das Lebenssystem im Master gemerged wurde.

smart